File: backend/src/main/java/com/gymapp/modules/training/dto/PlanExerciseResponse.java
package com.gymapp.modules.training.dto;

import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class PlanExerciseResponse {
    private Integer orderIndex;
    private String exerciseName;
    private String muscleGroup;
    private Integer sets;
    private Integer reps;
    private Integer restSeconds;
    private String notes;
}

File: android/app/src/main/java/com/gymapp/android/data/remote/response/PlanExerciseResponse.kt
package com.gymapp.android.data.remote.response

data class PlanExerciseResponse(
    val exerciseName: String,
    val muscleGroup: String?,
    val sets: Int,
    val reps: Int,
    val restSeconds: Int,
    val orderIndex: Int,
    val notes: String?
)